website/blog/2019-01-21-jest-24-new-website-babel-7-and-typescript-support.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Jest 24: New Website, Babel 7 and TypeScript support
```

Comment:
```suggestion
title: Jest 24: New Website, TODOs, Improved Assertions and default TypeScript support
```

Maybe?

Reply:
I agree that's better. I'll swap it around to "Jest 24: New Website, default TypeScript support, Improved Assertions and TODOs" as I think those are the more... visible changes I suppose is a word for it. thoughts?

Reply:
Yes, I think some readers might misunderstand the meaning of Babel 7

Reply:
CHanged in https://github.com/facebook/jest/pull/7670/commits/906b814eb80bf71a2bbffde0d8272c03bee113e6 but I think it could still be better